src/app/features/theme-drawer/theme-list-item.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from 'react';
+import { Box, BoxProps, color, Flex, Stack } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+import { useDrawers } from '@app/common/hooks/use-drawers';
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { UserSelectedTheme, useThemeSwitcher } from '@app/common/theme-provider';
+import { CheckmarkIcon } from '@app/components/icons/checkmark-icon';
+
+interface ThemeListItemProps extends BoxProps {
+  theme: UserSelectedTheme;
+}
+export const ThemeListItem = ({ theme, ...props }: ThemeListItemProps) => {
+  const { setIsShowingThemeSwitcher } = useDrawers();
+  const [_currentTheme, userSelectedTheme, setUserSelectedTheme] = useThemeSwitcher();
+  const isActive = theme === userSelectedTheme;
+  const analytics = useAnalytics();
+
+  const handleItemClick = useCallback(() => {
+    void analytics.track('theme_selected');
+    setUserSelectedTheme(theme);
+    setTimeout(() => setIsShowingThemeSwitcher(false), 25);
+  }, [analytics, setUserSelectedTheme, theme, setIsShowingThemeSwitcher]);
+
+  return (
+    <Box
+      width="100%"
+      key={theme}
+      _hover={
```

Comment:
Does prettier really format it like this?

Reply:
I believe so, the IDE I'm using seems to respect the prettier config, and the lint pipeline checks passed too.